aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -92,6 +92,12 @@
   public static final String GLUE_CATALOG_SKIP_ARCHIVE = "glue.skip-archive";
   public static final boolean GLUE_CATALOG_SKIP_ARCHIVE_DEFAULT = false;
 
+  /**
+   * If set, GlueCatalog will use Lake Formation for access control.
```

Comment:
we can add a link to the related credential vending documentation? And add a sentence that when enabled, the client factory must be `LakeFormationAwsClientFactory` or extend it as a base class.